sonar-db/src/main/java/org/sonar/db/user/GroupDao.java

```diff
@@ -44,19 +45,41 @@ public GroupDao(System2 system) {
     this.system = system;
   }
 
-  public GroupDto selectOrFailByName(DbSession session, String key) {
-    GroupDto group = selectByName(session, key);
+  /**
+   * @deprecated replaced by {@link #selectByName(DbSession, String, String)}
+   */
+  @Deprecated
+  public GroupDto selectOrFailByName(DbSession session, String name) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)